internal/settings/settings.go

```diff
@@ -371,16 +370,11 @@ func checkRuntimeParams(settings *Settings) error {
 		}
 	}
 
-	ballastSize := setMemoryBallast(memTotalSize)
-	memLimit, err := setMemoryLimit(memTotalSize)
+	_, err := setMemoryLimit(memTotalSize)
```

Comment:
`setMemoryLimit` should also be updated to not return a value if it's not needed anymore